### PR TITLE
Feat: Add Facebook Page Link Button on Landing Page

### PR DESCRIPTION
**Pull request by saenyakorn**

### CODE REVIEW

**Comment by saenyakorn** [comment]
Demo
Screen Shot 2564-10-10 at 21 44 50

File: components/announcements/recruiting.jsx [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://www.facebook.com/ThailandIncubator"
+                  className="bg-white box-border border-2 border-pink-600 px-4 py-2 rounded text-pink-600 text-xs tracking-widest flex items-center hover:bg-pink-100 transition duration-200 ease-in"
+                >
+                  VISIT <span className="ml-1 font-bold"> FACEBOOK PAGE</span>
+                </a>
+              </div>
+              <p className="text-sm text-gray-600 text-center font-normal leading-loose">
+                or search for "<span className="font-bold">Thinc. 9 Gen</span>" in your FB application!
```
Comment (bombnp) [inline components/announcements/recruiting.jsx:34]:
```suggestion
                or search for "<span className="font-bold">Thinc. 9th Gen</span>" in your FB application!
```

**Review by bombnp — COMMENTED** [review]
other than that, LGTM

File: components/announcements/recruiting.jsx (hunk 1 above)
Comment (saenyakorn) [reply components/announcements/recruiting.jsx:34]: next PR will do the dynamic generation with ordinal

**Pull request merged**